### PR TITLE
[WFLY-9122] Fixed Dwm servers not in manual mode

### PR DESCRIPTION
**Pull request by rjanik114**
https://issues.jboss.org/browse/WFLY-9122

The dwm servers are no longer always started and should not disrupt other tests.

Cc @yersan

### CODE REVIEW

**Comment by bstansberry** [comment]
this is ok to test

@maeste Please approve

**Comment by bstansberry** [comment]
The Windows failure is in the Dwm tests so I assume it is related.

**Comment by LittleJohnII** [comment]
It's definitely related. I'm looking into it, but it may take some time.

**Comment by kabir** [comment]
@LittleJohnII Please @mention me when done. I've added the fixme label to not have to keep looking at this PR until that happens

**Comment by LittleJohnII** [comment]
Finally got around to this.

@kabir the PR is fixed.

The issues on Windows were due to configuration from previous tests not being cleaned up properly. `File.renameTo(String)` fails on Windows if the target exists.

This time, the Linux run failed due to timeout. Is it reasonable to retest this?

**Review by stuartwdouglas — CHANGES_REQUESTED** [review]

File: testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/workmanager/distributed/AbstractDwmTestCase.java [hunk 1]
```diff
@@ -1,50 +1,36 @@
-/*
- * JBoss, Home of Professional Open Source.
- * Copyright 2017, Red Hat Middleware LLC, and individual contributors
- * as indicated by the @author tags. See the copyright.txt file in the
```
[the remark this answers is not in the document]
Comment (LittleJohnII) [reply testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/jca/workmanager/distributed/AbstractDwmTestCase.java]: That wasn't intentional. Fixed, thanks.

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**